config/initializers/ominiauth.rb

```diff
@@ -0,0 +1,14 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :developer if Rails.env.development?
+
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'],
+    {
+      scope: 'email, profile, http://gdata.youtube.com',
+      prompt: 'select_account',
+      image_aspect_ratio: 'square',
+      image_size: 50
+    }
+
+end
+
+OmniAuth.config.allowed_request_methods = %i[get post]
```

Comment:
Você está acessando a rota usando POST, então essa linha pode ser removida já que por padrão o método permitido é apenas POST.

Parece que usar GET permite CSRF, por isso, tem um warning quando GET está habilitado.

Tem mais informações sobre isso nas [realease notes do omniauth](https://github.com/omniauth/omniauth/releases/tag/v2.0.0)

```suggestion
OmniAuth.config.allowed_request_methods = %i[get post]
```

Reply:
Removi, faz sentido